TaxFoundation/itci-interactive
# Migrate from Mailchimp form to new Hubspot form.

Subscribe buttons now redirect to [the new hubspot form](https://taxfoundation.org/tax-newsletter/) on the main tax foundation website. 

### Steps to test
1. clone fork
2. run "npm install" and "npm run develop"
3. open in browser
4. click menu button to open menu and click orange subscribe link
5. exit out of menu, and scroll halfway down page
6. wait ~30 until email updates modal appears on bottom page and click 'Subscribe Today' button.
